server/src/services/organisation.js

```diff
@@ -133,6 +133,17 @@ const remove = async (id, db) => {
         .delete()
 }
 
+const hasUser = async (id, userId, knex) => {
+    const hasUser = await knex('user_organisation')
+        .select(1)
```

Comment:
I would use the native SQL count here instead of selecting 1 and check for result length.

http://knexjs.org/#Builder-count

Reply:
Why? To my knowledge this is more performant, and the way I've always done "checks" like these. Not that it really matters here though, but why use an aggregate function when you don't have to? I would rather select id or something and check the result length.

Reply:
Maybe you are thinking of count(1)? That's more performant.

Reply:
It also can come down to where it's used, consider you have a table with 50 million of rows, you wouldn't want to return 50 million ids and then count them in code

Reply:
I now see you use the limit(1) on the end as well so that would never been the case.